helm/content-store/values.yaml

```diff
@@ -1,47 +1,35 @@
-# Default values for content-store.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-deployment:
-  replicaCount: 1
-  name: content-store
-  image:
-    repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/content-store
-    version: latest
-  env:
-    - name: DEFAULT_TTL
-      value: "1800"
-    - name: GOVUK_APP_DOMAIN
-      value: "nadeemtest.svc.cluster.local"
-    - name: GOVUK_APP_DOMAIN_EXTERNAL
-      value: "default.test.govuk.digital"
-    - name: GOVUK_APP_NAME
-      value: "content-store-eks"
-    - name: GOVUK_APP_TYPE
-      value: "rack"
-    - name: GOVUK_CONTENT_SCHEMAS_PATH
-      value: "/govuk-content-schemas"
-    - name: GOVUK_ENVIRONMENT
-      value: "test"
-    - name: GOVUK_WEBSITE_ROOT
-      value: "https://www-origin.default.test.govuk.digital"
-    - name: GOVUK_WORKSPACE
-      value: "nadeem"
-    - name: MONGODB_URI
-      value: "mongodb://mongo-1.pink.test.govuk-internal.digital,mongo-2.pink.test.govuk-internal.digital,mongo-3.pink.test.govuk-internal.digital/content_store_production"
-    - name: PLEK_SERVICE_PUBLISHING_API_URI
-      value: "http://publishing-api-web.nadeemtest.svc.cluster.local"
-    - name: PLEK_SERVICE_ROUTER_API_URI
-      value: "http://router-api.nadeemtest.svc.cluster.local"
-    - name: PLEK_SERVICE_SIGNON_URI
-      value: "https://signon.nadeemtest.svc.cluster.local"
-    - name: PORT
-      value: "80"
-    - name: RAILS_ENV
-      value: "production"
-    - name: SECRET_KEY_BASE
-      value: ""
+# values used to compute other values
+govukEnvironment: test
+govukStack: pink
+govukDomainExternal: govuk.digital
+govukDomainInternal: govuk-internal.digital
+clusterDomain: svc.cluster.local
 
 service:
-  name: content-store-service
+  annotations: {}
+  type: NodePort
   port: 80
   targetPort: 80
+
+deployment:
+  replicaCount: 1
+  image:
+    repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/content-store
+    tag: latest
+  containerPort: 80
+  # values used to generate the environment variables for the pod
+  appDefaultTll: "1800"
+  appGovukAppDomain: #default (printf "%s.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.clusterDomain )
+  appGovukDomainExternal: #default (printf "%s.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.govukDomainExternal )
+  appGovukAppName: #default .Release.Name
+  appGovukAppType: "rack"
+  appGovukContentSchemasPath: "/govuk-content-schemas"
+  appGovukWebsiteRoot: #default (printf "https://www-origin.%s.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.govukDomainExternal )
```

Comment:
Does a null value here work? I thought you'd have to set an empty string with `""` to avoid YAML parse errors?